react-accordion/my-app/src/TopicCard.tsx

```diff
@@ -0,0 +1,24 @@
+type Topic = {
+  id: number;
+  title: string;
+  content: string;
+};
+type Props = {
+  topic: Topic;
+  activeId: number;
+  onTitleClick: (id: number) => void;
+};
+export function TopicCard({ topic, activeId, onTitleClick }: Props) {
+  return (
+    <>
+      <div className="title" onClick={() => onTitleClick(topic.id)}>
+        {topic.title}
+      </div>
+      {activeId === topic.id ? (
```

Comment:
This is a bit cleaner if you just use `{activeId === topic.id && <div>....</div>}` instead of the ternary ?:.